fsw/cfe-core/unit-test/es_UT.c

```diff
@@ -4797,12 +4798,22 @@ void TestAPI(void)
     CFE_ES_Global.AppTable[0].AppState = CFE_ES_AppState_RUNNING;
     CFE_ES_Global.AppTable[0].TaskInfo.MainTaskId = 15;
     OS_TaskCreate(&CFE_ES_Global.TaskTable[1].TaskId, NULL, NULL, NULL,
-                  0, 0, 0);
-    UT_Report(__FILE__, __LINE__,
-              CFE_ES_DeleteChildTask(CFE_ES_Global.TaskTable[1].TaskId) ==
-                CFE_SUCCESS,
+                  0, 0, 0);    
+    Return = CFE_ES_GetAppInfo(&AppInfo,CFE_ES_Global.TaskTable[1].AppId);
+    UtAssert_True(Return == CFE_SUCCESS, 
+         "CFE_ES_GetAppInfo() return=%x", (unsigned int)Return); 
+    UtAssert_True(AppInfo.NumOfChildTasks == 1, 
+         "AppInfo.NumOfChildTaskss == %u", (unsigned int)AppInfo.NumOfChildTasks); 
+    Return = CFE_ES_DeleteChildTask(CFE_ES_Global.TaskTable[1].TaskId);  
+    UtAssert_True(Return == CFE_SUCCESS, 
+         "DeleteChildResult() return=%x", (unsigned int)Return);
+    Return = CFE_ES_GetAppInfo(&AppInfo,CFE_ES_Global.TaskTable[1].AppId);
+    UtAssert_True(Return == CFE_SUCCESS, 
+         "CFE_ES_GetAppInfo() return=%x", (unsigned int)Return); 
+    UT_Report(__FILE__, __LINE__,
```

Comment:
All good up through here.  But why is this last case going back to using UT_Report?  It should look the same as line 4805, using UtAssert_True, just testing for a different expected value (0 instead of 1).

(UT_Report was just a compatibility wrapper for old test cases, and it doesn't allow context info to be logged)

Reply:
Resolving based on @jphickey comment below.